integration_tests/configs/nft_transfer.jsonnet

```diff
@@ -0,0 +1,36 @@
+local default = {
+  accounts: [
+    {
+      name: 'relayer',
+      coins: '100cro',
+    },
+    {
+      name: 'signer',
+      coins: '200cro',
+    },
+  ],
+  genesis: {
+    app_state: {
+      transfer: {
+        params: {
+          receive_enabled: true,
+          send_enabled: true,
+        },
+      },
+    },
+  },
+};
+local validator = {
+  coins: '10cro',
+  staked: '10cro',
+};
+
+{
```

Comment:
may I ask if nft transfter config is same as [ibc](https://github.com/crypto-org-chain/chain-main/blob/79e13c7df08494a835b1295d4292cbce80a732a7/integration_tests/configs/ibc.jsonnet) one?

Reply:
Removed unnecessary parts.